buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -169,6 +169,14 @@ def getPlatformAndLabel(SPEC) {
 			platformAndLabel[0] = 'x64_win'
 			platformAndLabel[1] = 'hw.arch.x86&&sw.os.win'
 			break
+		case ~/win_x86/:
+			platformAndLabel[0] = 'x64_win'
+			platformAndLabel[1] = 'hw.arch.x86&&sw.os.win'
+			break
+		case ~/zos_390-64.*/:
+			platformAndLabel[0] = 's390x_zos'
```

Comment:
`PLATFORM` is used to retrieve nightly/releases sdk. Do we have zOS sdk published? Note if not using personal build launch tests on Zos should be careful. That is user can only use custmized URL to download existing zOS sdk.

Reply:
You are right, we currently do not have an official nightly build of z/OS, so this was intended to only be used with customized, not nightly or released (in those latter 2 cases it will fail to find SDK, in much the same way as an openjdk9 will fail to find a release build, as there are none published, since they were cleared out).